services/catarse/config/locales/catarse_bootstrap/views/users/show.en.yml

```diff
@@ -76,6 +76,7 @@ en:
         account_type:
           conta_corrente: Checking Account
           conta_poupanca: Savings account
+          conta_facil: Easy Account
```

Comment:
[opinion]
O que seria esse Conta Fácil? Se for algo especifico do Brasil, eu acho que não deveria ser traduzido.